### ISSUE TITLE
Extended Industrialization

### ISSUE THREAD

**Issue by castilio20** [EnigmaticaModpacks/Enigmatica10#58]
### URL

https://www.curseforge.com/minecraft/mc-mods/extended-industrialization

### Why would you like the mod added?

an mi addon that may be worth checking out

**Comment by NielsPilgaard** [comment]
That is one large addon, could be worth looking into. @MuteTiefling have you seen this yet?

**Comment by MuteTiefling** [comment]
Yeah I've been keeping an eye on it. Haven't had a chance to test it, but it does look interesting.

**Comment by theboo** [comment]
They had me at ULTIMATE LASER DRILL

**Comment by MuteTiefling** [comment]
Been messing with it a bit this morning and it has some really nice features. It also has some really weird bugs (eg. https://github.com/Swedz/Extended-Industrialization/issues/7 and https://github.com/Swedz/Extended-Industrialization/issues/6). Hopeful they'll get fixed soon.

**Comment by Swedz** [comment]
Both issues are fixed in 1.4.1, thanks for reporting them :) If there's any other issues let me know and I'll fix them asap

**Comment by MuteTiefling** [comment]
Awesome thanks!
